src/_layouts/post.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {% include head.html %}
+
+  <body>
+    {% include header.html %}
+
+    <main>
+      <div id="main_inner">
```

Comment:
We probably should have a 'Back to blog posts' at the start and/or end of this- maybe just the end so that someone who reads to the bottom can immediately hop back to see more (when there are more).

Reply:
Will look into doing this.

Reply:
Has been added.